src/components/UI/Header/ImageContainer.styled.js

```diff
@@ -1,9 +1,9 @@
 import styled from 'styled-components';
 
 const ImageContainer = styled.div`
-	width: 35%;
+	width: 28.5%;
```

Comment:
Again, I'd probably suggest defining a variable for this. :)

Reply:
I changed it!